### DISCUSSION TITLE
Support for Exception.Data?

### DISCUSSION THREAD

**Discussion by ShaneCourtrille** [RehanSaeed/Serilog.Exceptions#11] [Q&A]
Is there support for logging of the key/value pairs in the [Exception.Data](https://msdn.microsoft.com/en-us/library/system.exception.data%28v=vs.110%29.aspx) property?

**Comment by RehanSaeed** [comment]
Yes

**Comment by ShaneCourtrille** [comment]
Ahhh I see it in the example now.  Thanks.
